core/src/main/java/org/mule/runtime/core/api/Event.java

```diff
@@ -502,6 +503,10 @@ static Event getCurrentEvent() {
    */
   static void setCurrentEvent(Event event) {
     CurrentEventHolder.currentEvent.set(event);
+    if (event != null) {
+      ThreadContext.put("correlationId", event.getCorrelationId());
+      ThreadContext.put("originatingFlowName", event.getContext().getOriginatingFlowName());
```

Comment:
this is no longer available. get the flowName form the component location in the context